storage/innobase/xtrabackup/test/t/ib_external_xa_prepared.sh

```diff
@@ -0,0 +1,42 @@
+############################################################################
+# Test external XA prepare transactions in backup result are not rolled back
+# after --prepare
+############################################################################
+
+. inc/common.sh
+
+vlog "start server with gtid enabled"
+start_server --gtid-mode=ON --enforce-gtid-consistency=ON --log-bin --log-slave-updates
+
+run_cmd $MYSQL $MYSQL_ARGS test -e "create table t1(id int, primary key(id)) engine=innodb"
+run_cmd $MYSQL $MYSQL_ARGS test -e "xa start 'test1'; insert t1 values (1); xa end 'test1'; xa prepare 'test1'"
+run_cmd $MYSQL $MYSQL_ARGS test -e "xa start 'test2'; insert t1 values (2); update t1 set id = 3 where id = 2; xa end 'test2'; xa prepare 'test2'"
+
+run_cmd $MYSQL $MYSQL_ARGS test -e "xa recover" | grep "test1"
+run_cmd $MYSQL $MYSQL_ARGS test -e "xa recover" | grep "test2"
+
+vlog "Make full backup"
+mkdir $topdir/backup
+innobackupex --slave-info --no-timestamp $topdir/backup
```

Comment:
Because `innobackupex` is removed in 8.0, it is better to use `xtrabackup` in new test cases. I assume this test case is passing on 8.0?

Reply:
I have not tried 8.0, and I guess it will fail.
I'll change to use `xtrabackup`, and try 8.0.